common/journal_entries/imperia_true_neutral_journal_entries.txt

```diff
@@ -3,16 +3,43 @@
 	group = je_group_foreign_affairs
 	on_weekly_pulse = {
 		effect = {
+			set_local_variable = {
+				name = true_neutral_applicant_signee_list_counter
+				value = 0
+			}
 			every_in_list = {
 				variable = true_neutral_applicant_signee_list
 				imperia_update_unsigned = yes
+				# Start counting actual countries left
+				change_local_variable = {
+					name = true_neutral_applicant_signee_list_counter
+					add = 1
+				}
+			}
+			# Kinda dirty fix, since we cannot check for the existance of dynamic, non-specific countries
+			if = {
+				limit = {
+					local_var:true_neutral_applicant_signee_list_counter < imperia_true_neutral_remaining_signers_value
+				}
+				# If theres more remaining signers (on paper) than there are actual existing countries on the list, reduce the list by 1
+				change_variable = {
+					name = true_neutral_required_signers_var
+					subtract = 1
+				}
```

Comment:
Why tick this down every week? Why not just set true_neutral_required_signers_var to local_var:true_neutral_applicant_signee_list_counter?